great_expectations/render/renderer/site_builder.py

```diff
@@ -403,6 +423,10 @@ def build(self):
                     )
                 expectation_suite_name = key_resource_identifier.expectation_suite_identifier.expectation_suite_name
                 run_id = key_resource_identifier.run_id
+                if run_id == "profiling":
```

Comment:
Agreed. Since we have the same issue with the hard-coded expectation_store name at this point (and an issue to track, I think we consider this resolved and move on.